data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+
+Marking links as outlinks in HTML code
+""""""""""""""""""""""""""""""""""""""
+
+Similar as downloads, links can be set to be treated as outlinks manually, but only with CSS classes, you cannot use a HTML attribute.
+
+You can use one of default CSS classes: `piwik_link` or `piwik-link`. eg. 
+
+```
+<a href='https://piwik.pro' class="piwik-link">Piwik PRO</a>
+```
+
+Or you can define your custom CSS classes for outlinks with ours :ref:`Javascript Tracker API<force-tracking-using-css-class>`. 
+
+```
+// now all clicks on links with the css class "external" will be counted as outlinks
+
+// you can also pass an array of strings
+_paq.push(['setLinkClasses', "custom-link-class"]);
+
+_paq.push(['trackPageView']);
+```
+
+or a list of classes
+
+```
+_paq.push(['setLinkClasses', ["custom-link-class", "other-link-class"]]);
+
+_paq.push(['trackPageView']);
+```
+
+and in HTML code
+
+```
+<a href='https://piwik.pro' class="custom-link-class">Piwik PRO</a>
+```
+
+Marking links with inline Javascript
+""""""""""""""""""""""""""""""""""""
+
+Alternatively you can use an inline javascript and onclick attribute to track any link as an outlink.
+
+```
+<a href="mailto:support@piwik.pro" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/support-contact-form', 'link']);">Write us a message.</a>
+```
+
+
+Other related  abilities
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Changing delay for link tracking
+""""""""""""""""""""""""""""""""
+All link tracking uses a slight delay of click execution, so the browser won't exit the page before a click is tracked. The default value of such delay is 500ms, but you can modify it as you wish. You have to remember that if you set this value too low, it might be not enough to track the click, if you set it too high, the browser might ignore the delay.
+
+```
+_paq.push(['setLinkTrackingTimer', 300]); // 300 milliseconds
+
+_paq.push(['trackPageView']);
+```
+
+Disable download and outlink tracking
+"""""""""""""""""""""""""""""""""""""
+
+To explicitly disable link tracking you can use `disableLinkTracking` function. After sing it, all of link clicks won't be tracked.
+
+```
+_paq.push(['disableLinkTracking']);
+```
+
+Disabling link tracking with CSS classes
+""""""""""""""""""""""""""""""""""""""""
+
+You can mark links that you do not with to track with CSS classes. JS Tracker will ignore such links and won't track them.
+
+```
+_paq.push(['setIgnoreClasses', "do-not-track"]);
+_paq.push(['trackPageView']);
+```
+
+or a list of classes:
+
+```
+_paq.push(['setIgnoreClasses', ["dont-track-this", "this-either", "nor-this"]]);
+_paq.push(['trackPageView']);
+```
+
+and later in HTML code:
+
+```
+<a href="https://piwik.pro/document.pdf" class="dont-track-this">A document, that should not be tracked.</a>
+```
+
+A Tip
+"""""
+
+To increase accuracy of download and outlink tracking, you can consider enabling the use of :ref:`navigator.sendBeacon()<navigation-send-beacon>`.
```

Comment:
We should consider setting is by default for download and outlinks if it is a valid tip. I think this tip would need a longer description and a different name. Maybe remove it here and add a task for it?

Reply:
Ok, I will do so.

Reply:
https://piwikpro.atlassian.net/browse/PPTT-2763

Reply:
thanks